config/params.php

```diff
@@ -44,11 +54,26 @@
         ],
     ],
 
+    'yiisoft/mailer' => [
+        'composerPath' => '@resources/mail',
+        'writeToFiles' => true,
+        'writeToFilesPath' => '@runtime/mail',
+        'swiftSmtpTransport' => [
+            'host' => 'smtp.example.com',
+            'port' => 25,
+            'encryption' => null,
+            'username' => 'admin@example.com',
+            'password' => ''
+        ],
+    ],
+
     'yiisoft/view' => [
         'defaultParameters' => [
             'applicationParameters' => ApplicationParameters::class,
             'assetManager' => AssetManager::class,
+            'field' => Field::class,
```

Comment:
What's the advantage compared to using `Field::widget()` directly in the view?

Reply:
To show how to configure a widget globally for use.